components/ThemeToggler.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div class="theme-toggle-wrapper" @click="toggleTheme()">
+    <span>
+      <img class="theme-toggle-icon" :src="require(`~/assets/${themeImage}`)" />
+    </span>
+  </div>
+</template>
+
+<script>
+function setCookie(name, value, days) {
```

Comment:
NIT: What is your opinion of keeping `setCookie` in a utility function

Reply:
Right :100: